zmq-async/src/jbuild

```diff
@@ -4,5 +4,4 @@
  ((name zmq_async)
   (public_name zmq-async)
   (wrapped false)
```

Comment:
`(wrapped false)` should be omitted here.

Reply:
That would mean that Zmq_async should be referenced though
`Zmq_async.Zmq_async`? Or am I wrong?

If we don't wrap, we should make sure that all modules inside are prefixed with zmq_ to avoid clashes with other modules.

Reply:
> That would mean that Zmq_async should be referenced though
Zmq_async.Zmq_async? Or am I wrong?

Not quite.

If you write a `zmq_async.ml` module, then this will be your entry level module for the library. All constituting modules will be accessible with their shortened names. For example, in the current setup, the `Deferred` module would be properly private and only available as `Zmq_async__Deferred` (as we'd like). If you'd like to make the Deferred module accessible to the user, you may include a `module Deferred = Deferred` line in your `Zmq_async` module. If you omit `Zmq_async.ml` then jbuilder will simply generate one for you with all the modules in your library aliases as:

```ocaml
module Deferred = Deferred (* the compilation unit actually Zmq_async__Deferred *)
module Foo = Foo
(* etc. *)
```

> If we don't wrap, we should make sure that all modules inside are prefixed with zmq_ to avoid clashes with other modules.

This is right. But note that `(wrapped false)` should only be used for single module libraries and only exists in jbuilder for compatibility.

Reply:
Thats great -  I did not know that. 
Re-reading the paragraph on wrapped does actually say it, but I completely missed it previously. 
Ill change asap.

Reply:
I have wrapped the modules (removed the line), but now I cannot access Deferred. I'm accessing it though its internal (wrapped) name but its not nice. I could expose it, but I really like the idea of hiding it from users. An alternative would be to copy deferred.ml (though copy stanza) into the test dir. Suggestions?